src/Runtime/ExecutionSession.cpp

```diff
@@ -96,10 +92,12 @@ OMTensorList *ExecutionSession::run(OMTensorList *input) {
   return _entryPointFunc(input);
 }
 
-std::string ExecutionSession::inputSignature() { return _inputSignatureFunc(); }
+std::string ExecutionSession::inputSignature() {
```

Comment:
ExecutionSession::inputSignature() --> ExecutionSession::inputSignature() const. Similar comment for `ExecutionSession::outputSignature()`